deploy-tool/main.go

```diff
@@ -142,10 +144,25 @@ func generateAndDeployKeyPair(namespace, keyName string) (*ecdsa.PrivateKey, err
 	return ecdsaKey, nil
 }
 
+// getConfigFilePath gets the configuration file path from the DEPLOY_CONFIG_PATH environment variable.
+// this function defaults to ./config.toml if that env variable is empty.
+func getConfigFilePath() string {
+	val := os.Getenv("DEPLOY_CONFIG_PATH")
+	if len(val) == 0 {
+		return "./config.toml"
```

Comment:
Help me understand where the config file comes from. I don't think any of the values in there are emitted by prior stages of the deployment so I don't think it's emitted from the Terraform module. Looking at what's defined in there, and keeping in mind that imminently we'll be using Google Cloud DNS, it seems like the point of the config file is to allow overriding the default behavior, which would be to use the production Let's Encrypt, use Google Cloud DNS, use Google Cloud Project API credentials from `~/.config/gcloud`, and the Kubernetes driver. I think that's useful for testing this, but now I wonder if in order to absolve every deployer of `prio-server` from being required to make a well-formatted `config.toml`, `deploy-tool/config/config.go` should define default values for the common case.

Reply:
The config is going to be something the person deploying needs to configure. I'll add a few more defaults but at the very least a few things need to be manually entered:

- Assuming DNS provider defaults to GCP, we'd need the `[dns.gcp]` section (when that gets built out).
- The `acme.email, acme.subscriber_agreement`.
- Defaulting `storage.driver` to `kubernetes` and `storage.kubernetes.namespace` to `prio-server`

Reply:
I created `enpa-prio-ops@letsencrypt.org` yesterday for another purpose. Can we set that as the default email?